roqoqo-qasm/src/interface.rs

```diff
@@ -929,9 +932,13 @@ pub fn gate_definition(operation: &Operation) -> Result<String, RoqoqoBackendErr
         Operation::Hadamard(_) => Ok(String::from(
             "gate h a { u2(0,pi) a; }"
         )),
-        Operation::CNOT(_) => Ok(String::from(
+        Operation::CNOT(_) => match qasm_version {
+            QasmVersion::V2point0 => Ok(String::from(
             "gate cx c,t { CX c,t; }"
         )),
+        QasmVersion::V3point0(_) => Ok(String::from(
+            "gate cx c,t { ctrl @ x c,t; }"
+        ))},
```

Comment:
If this is braket-specific, I would make it so that it is only changed for that dialect.